src/html/fetch-content.js

```diff
@@ -22,8 +22,9 @@ async function fetchContent(context, {
   }
   const { content } = context;
   const {
-    owner, repo, path, branch = 'master', ref,
+    owner, repo, path, ref,
   } = request.params;
+  const branchOrRef = request.params.branch || ref || 'master';
```

Comment:
Are you sure about `request.params.branch`?

Reply:
pretty much... the schema says: `branch` defaults to `ref` defaults to `master`.
so if a `branch` is provided, it is preferred.

without this, this test failed: https://github.com/adobe/helix-pipeline/blob/master/test/testHTML.js#L646


IMO, we should get rid of `branch` and just use `ref` everywhere.

Reply:
I think the reason we introduced `branch` was to enable invalidations. And that means that the new source URL is not sufficient.

If branch `publish` has a current ref `…2cba` and gets a content update, then we need to invalidate hashes that include `publish`, but hashes for `…2cba` can stay the same.

So I'd say this PR is probably wrong.

Reply:
ok. I fixed it.